templates/Dockerfile.template

```
@@ -0,0 +1,24 @@
+FROM {{ base_image }}
+
+ENV HOME=/home/ops \
+    CONDA_DIR=/opt/conda
+
+USER root
+RUN $CONDA_DIR/bin/conda install -y nb_conda_kernels
+
+USER ops
+RUN $HOME/verdi/bin/pip install 'ipykernel<5.0.0'
```

Should be moved above under line 7 and modified to:
```suggestion
RUN $CONDA_DIR/bin/pip install 'ipykernel<5.0.0'
```